docs/core/experiments/manager.md

```diff
@@ -0,0 +1,74 @@
+# Experiments Manager
+
```

Comment:
Maybe change the PR name from "Docs: Experiments Manager [ED-4464]"  to "New: Docs experiments Manager [ED-4464]"

Reply:
Done.